src/config/general.h

```diff
@@ -161,16 +161,32 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 //#define PCI_CMD		/* PCI commands */
 //#define PARAM_CMD		/* Request parameter commands */
 //#define NEIGHBOUR_CMD		/* Neighbour management commands */
-//#define PING_CMD		/* Ping command */
+#define PING_CMD		/* Ping command */
 //#define CONSOLE_CMD		/* Console command */
 //#define IPSTAT_CMD		/* IP statistics commands */
 //#define PROFSTAT_CMD		/* Profiling commands */
 //#define NTP_CMD		/* NTP commands */
-//#define CERT_CMD		/* Certificate management commands */
+#define CERT_CMD		/* Certificate management commands */
 //#define IMAGE_MEM_CMD		/* Read memory command */
 #define IMAGE_ARCHIVE_CMD	/* Archive image management commands */
 #define SHIM_CMD		/* EFI shim command (or dummy command) */
 
+/*
+ * DHCP-specific options
+ *
+ */
+#undef DHCP_DISC_START_TIMEOUT_SEC
+#define DHCP_DISC_START_TIMEOUT_SEC	4
+
+#undef DHCP_DISC_END_TIMEOUT_SEC
+#define DHCP_DISC_END_TIMEOUT_SEC	32
+
+#undef DHCP_DISC_MAX_DEFERRALS
+#define DHCP_DISC_MAX_DEFERRALS		180
+
+#undef LINK_WAIT_TIMEOUT
+#define LINK_WAIT_TIMEOUT ( 45 * TICKS_PER_SEC )
```

Comment:
MTL-1957 pulled the `LINK_WAIT_TIMEOUT` out by request of CASMNET it never made it into CMS-iPXE:
- PR removing it in metal-ipxe https://github.com/Cray-HPE/ipxe/pull/7/files
- Currently living in ipxe-tpsw-clone: https://github.com/Cray-HPE/ipxe-tpsw-clone/blob/develop/src/config/general.h#L171-L172

I suppose it makes sense to keep this diverged so that this PR isn't turning too many knobs.